ui/components/app/transaction-decoding/components/ui/copy-raw-data/index.scss

```diff
@@ -0,0 +1,30 @@
+.copy-raw-data {
+  display: flex;
+  align-items: center;
+
+  &__button {
+    border-radius: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 8px;
+    margin-top: 5px;
+    background-color: transparent;
+
+    &:hover {
+      background-color: $Grey-000;
+    }
+
+    &:active {
+      background-color: #ededed;
+    }
+  }
+
+  &__icon {
+    padding-right: 6px;
```

Comment:
`padding-inline-end` would be more friend to RTL.

Reply:
1efaa13e6